kcidb/monitor/spool/__init__.py

```diff
@@ -97,6 +97,12 @@ def post(self, notification, timestamp=None):
             timestamp:      An "aware" datetime.datetime object specifying the
                             notification creation time, or None to use
                             datetime.datetime.now(datetime.timezone.utc).
+            time_to_send:   An "aware" datetime.datetime object specifying the
```

Comment:
Instead of `time_to_send` can we call this `due`, here and everywhere else? That would be shorter.